docs/src/content/docs/ja/guides/sidebar.mdx

```diff
@@ -0,0 +1,467 @@
+---
+title: サイドバーのナビゲーション
+description: Starlightサイトのサイドバーのナビゲーションリンクを設定・カスタマイズする方法を学びます。
+sidebar:
+  badge: 新規
+---
+
+import FileTree from '../../../../components/file-tree.astro';
+import SidebarPreview from '../../../../components/sidebar-preview.astro';
+
+サイドバーはユーザーがサイト内を移動するための主要な方法の1つであるため、サイドバーを整理することは良いドキュメントの鍵となります。Starlightでは、サイドバーのレイアウトとコンテンツをカスタマイズするためのオプション一式を提供しています。
+
+## デフォルトのサイドバー
+
+Starlightはデフォルトで、ドキュメントのファイルシステム構造に基づいてサイドバーを自動的に生成します。その際、各ファイルの`title`プロパティをサイドバーのエントリとして使用します。
+
+たとえば以下のファイル構造があるとします。
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+      - reference/
+        - configuration.md
+
+</FileTree>
+
+すると以下のサイドバーが自動的に生成されます。
+
+<SidebarPreview
+	config={[
+		{
+			label: 'guides',
+			items: [
+				{ label: 'Components', link: '/guides/components/' },
+				{ label: 'Internationalization (i18n)', link: '/guides/i18n/' },
+			],
+		},
+		{
+			label: 'reference',
+			items: [
+				{ label: 'Configuration Reference', link: '/reference/configuration/' },
+			],
+		},
+	]}
+/>
+
+自動生成されるサイドバーについては、[自動生成されるグループ](#自動生成されるグループ)のセクションで詳しく説明します。
+
+## リンクとリンクグループを追加する
+
+サイドバーの[リンク](#リンク)と（折りたたみ可能なヘッダー内の）[リンクグループ](#グループ)を設定するには、`astro.config.mjs`で[`starlight.sidebar`](/ja/reference/configuration/#sidebar)プロパティを使用します。
+
+リンクとグループを組み合わせることで、さまざまなサイドバーレイアウトを作成できます。
+
+### リンク
+
+`label`と`link`プロパティをもつオブジェクトを使用して、内部または外部ページへのリンクを追加します。
+
+```js
+starlight({
+	sidebar: [
+		// CSSとスタイリングガイドへのリンク。
+		{ label: 'CSSとスタイリング', link: '/ja/guides/css-and-tailwind/' },
```

Comment:
I'm not 100% sure how links in examples should be handled. However, I looked at the italian translation and found that all examples link to englisch content. Therefore, I _think_ you need to make this little improvement on every sidebar example:

```suggestion
		{ label: 'CSSとスタイリング', link: '/guides/css-and-tailwind/' },
```